appgate/state.py

```diff
@@ -262,12 +262,13 @@ async def plan_apply(plan: Plan, namespace: str, k8s_configmap_client: K8SConfig
     for e in plan.create.entities:
         log.info('[appgate-operator/%s] + %s %s [%s]', namespace, type(e.value), e.name, e.id)
         if entity_client:
-            if not await entity_client.post(e.value):
-                errors.add(e.id)
-            else:
+            try:
+                await entity_client.post(e.value)
                 name = 'singleton' if e.value._entity_metadata.get('singleton', False) else e.name
                 await k8s_configmap_client.update(key=entity_unique_id(e.value.__class__.__name__, name),
                                                   generation=e.value.appgate_metadata.current_generation)
+            except Exception:
+                errors.add(e.id)
```

Comment:
We should log the error though.

Reply:
https://github.com/appgate/sdp-operator/issues/60 Created a task